src/hostapi/wasapi/pa_win_wasapi.c

```diff
@@ -87,6 +87,10 @@
     #include <mmdeviceapi.h>
     #include <devicetopology.h>    // Used to get IKsJackDescription interface
     #undef INITGUID
+//Visual Studio 2010 does not like the inline keyword
```

Comment:
Would you please add space after the // and modify `does not like the inline keyword` to `does not support the inline keyword`. After this change all will be ok and will be able to merge it.

Reply:
done as requested